lib/src/di/injector.dart

```diff
@@ -164,20 +166,18 @@ void _injectNotifiers() {
         AuthenticationMessageProvider(),
         FieldErrorMessageProvider(),
       ));
-  _locator.registerFactory<PopularNewsCriterionMessageMapper>(
-      () => PopularNewsCriterionMessageLocalMapper());
+  _locator.registerFactory<PopularNewsCriterionMessageMapper>(() => PopularNewsCriterionMessageLocalMapper());
```

Comment:
zmieniłeś line length ze 100 na więcej? :P

Reply:
nie wiem jak to się stało, ale miałem 120, zmienię z powrotem na 100